### ISSUE TITLE
Add multi-language support

### ISSUE THREAD

**Issue by goldengrape** [clmnin/summarize.site#3]
Just translate "Give me the summary of" to the target language 
```
gptQuestion = `Give me the summary of :\n\n${request.content}`
```

to Chinese:
https://translate.google.com/?sl=auto&text=Give%20me%20the%20summary%20of&op=translate&tl=zh-CN
to Spanish_language
https://translate.google.com/?sl=auto&text=Give%20me%20the%20summary%20of&op=translate&tl=es
....
ISO 639-1 codes list in: 
https://en.wikipedia.org/wiki/ISO_639-1

**Comment by krlvi** [comment]
here's a PR adding a config for the prompt to use https://github.com/clmnin/summarize.site/pull/5 
This is a flexible approach but not sure if it's the preferred one by @clmnin

**Comment by clmnin** [comment]
@goldengrape I've merged PR #5 . Let us know if this works for you.

**Comment by wenlzhang** [comment]
The following two configurations work for me:

- Rewrite this for brevity, translate into Chinese, in outline form:
    - This directly summarizes the article in Chinese, outline form.
- Rewrite this for brevity, in outline form, Chinese translation:
    - This summarizes the article in outline form in English, and adds a Chinese translation below.

**Comment by clmnin** [comment]
@wenlzhang I've updated the README (added an FAQ) to reflect your prompt.

**Comment by clmnin** [comment]
This issue was fixed in 1832926cbf4909acff8008da6eba8c2005044749
